tests/magnetostatics/test_circuits.py

```diff
@@ -314,3 +435,77 @@ def test_cariddi(self):
         plt.close(fig)
 
         assert np.max(np.abs(ripple - self.cariddi_ripple)) < 0.04
+
+
+class TestPolyhedral2DRing:
+    @classmethod
+    def setup_class(cls):
+        radius = 7
+        n = 50
+        ring = make_circle(radius, [0, 0, 0], 0, 360, [0, 1, 0])
```

Comment:
Best to change this test so that the axis of the ring is the z-axis, so that you can compare with 2-D semi-analytical formulations that have already been benchmarked.